tests/integration/api_v3/rpc/blocks.assets.test.js

```diff
@@ -0,0 +1,318 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import moment from 'moment';
+
+const config = require('../../../config');
+
+const {
+	request,
+} = require('../../../helpers/socketIoRpcRequest');
+
+const {
+	invalidParamsSchema,
+	emptyResultEnvelopeSchema,
+	emptyResponseSchema,
+	jsonRpcEnvelopeSchema,
+	metaSchema,
+} = require('../../../schemas/rpcGenerics.schema');
+
+const {
+	blockAssetSchema,
+} = require('../../../schemas/api_v3/block.schema');
+
+const wsRpcUrl = `${config.SERVICE_ENDPOINT}/rpc-v3`;
+const getBlocksAssets = async params => request(wsRpcUrl, 'get.blocks.assets', params);
+
+describe('Method get.blocks.assets', () => {
+	let refBlockAsset;
+	beforeAll(async () => {
+		[refBlockAsset] = (await getBlocksAssets({ limit: 1, offset: 5 })).result.data;
+	});
+
+	describe('is able to retireve block assets', () => {
+		it('no params -> ok', async () => {
```

Comment:
Please ensure the changes suggested/requested in `tests/integration/api_v3/http/blocks.assets.test.js` are reflected in this file as well.